base/workflow/R/run.write.configs.R

```diff
@@ -211,13 +228,26 @@ run.write.configs <- function(settings, ensemble.size, input_design, write = TRU
   PEcAn.logger::logger.info("###### Finished writing model run config files #####")
   PEcAn.logger::logger.info("config files samples in ", file.path(settings$outdir, "run"))
 
-  ### Save output from SA/Ensemble runs
-  # A lot of this is duplicate with the ensemble/sa specific output above, but kept for backwards compatibility.
-  save(ensemble.samples, trait.samples, sa.samples, runs.samples, pft.names, trait.names,
-    file = file.path(settings$outdir, "samples.Rdata")
-  )
-  PEcAn.logger::logger.info("parameter values for runs in ", file.path(settings$outdir, "samples.RData"))
+  # write runs manifest
+
+  manifest.file <- file.path(settings$outdir, "runs_manifest.csv")
+
+  if (nrow(run_manifest_df) > 0) {
```

Comment:
Is it OK for the manifest to be empty? If so I'd favor writing a zero-row file so it's easy to tell that afterwards; if not OK then should we throw an error here instead of skipping?